custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py

```diff
@@ -1319,17 +1612,84 @@ def match_facility(domain, xlsx_facility_name):
 def match_phi(domain, xlsx_phi_name):
     location_name, location_id = match_location(domain, xlsx_phi_name, "phi")
     if not location_id:
-        raise Exception("A valid phi is required")
+        raise ValidationFailure("A valid phi is required")
     return location_name, location_id
 
 
 def get_tu(domain, phi_id):
     if not phi_id:
         return None, None
-    phi = SQLLocation.get(domain=domain, location_id=phi_id)
+    phi = SQLLocation.active_objects.get(domain=domain, location_id=phi_id)
     return phi.parent.name, phi.parent.location_id
 
 
+def get_drtb_hiv_location(domain, district_id):
+    if not district_id:
+        return None, None
+    drtb_hiv = SQLLocation.get(domain=domain, parent__location_id=district_id, location_type__code="drtb-hiv")
+    return drtb_hiv.name, drtb_hiv.location_id
+
+
+class PersonIdGenerator(object):
+
+    dry_run_counter = 0
```

Comment:
This is interpreter-level global state, which seems like not a great thing. Can you justify that? I suppose since it's part of a management command it's not a big deal?